docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -35,6 +35,10 @@ This menu offers security-sensitive options like:
   problems with DMA/IOMMU during OS boot, keep it disabled.
   NOTE: Windows may not boot if the option is enabled.
 
+- `Enable Wi-Fi + BT radios` - (applicable to devices with Wi-Fi and Bluetooth
+  cards) - Enables or disables radios for Wi-Fi and Bluetooth. When disabled,
+  the slot is unconfigured and power to the wireless module is disconnected.
```

Comment:
```suggestion
- `Enable Wi-Fi + BT radios` - (applicable to laptops) - Enables or disables
  radios for Wi-Fi and Bluetooth. When disabled, the slot is unconfigured and
  power to the wireless module is disconnected.
```

Desktops typically have multiple PCIe slots, so the card may basically be in any slot and it is not always possible to cut off the power from it. Also for Protectli devices, it is not possible to cut off power, despite the WiFi slot has fixed location/Root Port.

Reply:
Suggestion applied in f888a04cad7509319e4a4cb0b8b7605ef959d268